tests/test_cli_layout_focus_contract.py

```
@@ -135,6 +158,24 @@ def run_cli(
     return proc.stdout.strip()
 
 
+def run_cli_without_socket(cli: str, args: list[str]) -> str:
+    env = dict(os.environ)
+    for key in ["CMUX_SOCKET_PATH", "CMUX_SOCKET", "CMUX_WORKSPACE_ID", "CMUX_SURFACE_ID", "CMUX_TAB_ID"]:
+        env.pop(key, None)
+    proc = subprocess.run(
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)